docs/cloud/guides/troubleshoot.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: "Troubleshooting"
+---
+
+## Columns could not be loaded
+
+This error can happen because of a few reasons:
+
+1. check that your elementary dbt package version is 0.12.0 or higher
```

Comment:
@ellakz 
0.12.0?

Reply:
@haritamar I think you added a fallback for old versions, right?